test/specs/ellipse.spec.js

```diff
@@ -34,12 +34,12 @@ describe('Ellipse annotation', function() {
               y: center.y + Math.sin(rad) * (yRadius + halfBorder)
             }, center, rotation / 180 * Math.PI);
 
-            expect(element.inRange(x, y)).withContext(`in, rotation: ${rotation}, angle: ${angle}, borderWidth: ${borderWidth}, {x: ${x.toFixed(1)}, y: ${y.toFixed(1)}}`).toEqual(true);
```

Comment:
No, I did not mean removing the whole context, just the `in` or `out` word. Those numbers in an failure message are really helpful when something fails.

Reply:
done